src/nbl/video/IQueue.cpp

```diff
@@ -3,6 +3,10 @@
 #include "nbl/video/ILogicalDevice.h"
 #include "nbl/video/TimelineEventHandlers.h"
 
+#include "git_info.h"
```

Comment:
why is `git_info` on its own weird path + why its header guard is `__GIT_INFO_H_INCLUDED__` and not `_NBL_GIT_INFO_H_INCLUDED_`